blockstack/blockstackd.py

```diff
@@ -3754,6 +3754,10 @@ class Program {
             namespace['lifetime'] = 0 if namespace_info['lifetime'] == NAMESPACE_LIFE_INFINITE else namespace_info['lifetime']
             if namespace_id in namespace_patches:
                 namespace.update(namespace_patches[namespace_id])
+            # adjust price coeff for version 1 (and version 2 that are more than a year old)
+            elif namespace_info['version'] == 1 or namespace_info['ready_block'] < 611400:
```

Comment:
This should be:

```suggestion
            elif namespace_info['version'] == 1 or (namespace_info['version'] == 2 and namespace_info['ready_block'] < 611400):
```

That is, this scaling should only apply to v1 and v2 namespaces.

Reply:
Very explicit, very cool